packages/pelagos/src/components/Dialog.js

```diff
@@ -45,16 +45,14 @@ const Dialog = ({id, className, title, role, size, stretch, initialFocus, childr
 		<div className="Dialog__backdrop">
 			{onSubmit ? (
```

Comment:
Not a big deal, but this could be moved to within the children of the `<Layer>` because they are now identical:
```js
<Layer
  id={id}
  className={fullClassName}
  level={1}
  role={role}
  aria-modal
  aria-labelledby="dialogTitle"
  ref={element}>
    {onSubmit ? (<form onSubmit={handleSubmit}>{content}</form>) : content}
</Layer>
```

Reply:
You're absolutely right, thank you!